### PR TITLE
Increase max log file size; increase message persist to disk size

### PR DESCRIPTION
**Pull request by jkosternl**
Info: The current defaults are outdated. Sergi experienced already the disadvantage of the current settings, to lose important log files. 10MB is a better default, which is easy to load in text editors or other tools. 

Since heap sizes are not 256MB anymore nowadays, it is a good idea to move to 2MB message to persist to disk, instead of 512KB. This increases performance, since disks are slow and memory is fast (and GC algorithms are fast and parallel as well).

Summary:
- Increase max log file size to 10MB by default; 
- increase message persist to disk size

### CODE REVIEW

**Comment by nielsm5** [comment]
Increasing message persistence to disk is ok, I think even 5 mb is reasonable. However the log files configuration should not change. Whoever has told you they are having rollover problems should either set these properties them selves or turn of DEBUG logging in a containerized environment (use at least INFO !).

**Comment by jkosternl** [comment]
> Increasing message persistence to disk is ok, I think even 5 mb is reasonable. However the log files configuration should not change. Whoever has told you they are having rollover problems should either set these properties them selves or turn of DEBUG logging in a containerized environment (use at least INFO !).

Sure, I'll make the persistence limit 5MB.

Nobody told me there are problems with rolling log files. But in this decade, it is no issue to transfer, open, search in log files of 10MB. And if something happens (errors & stacktraces) the logs rotates too fast and you will lose important information. Even when you have configured the log level at INFO.

File: core/src/main/resources/AppConstants.properties [hunk 1]
```diff
@@ -411,8 +411,8 @@ ibistesttool.maxMessageLength=1000000
 ibistesttool.reportTransformation=TestTool/xsl/Default.xsl
 # Maximum database size. When necessary the oldest report(s) will be deleted when adding a new one (default is 100MB)
 #   ibistesttool.maxStorageSize=100MB
-# Maximum file size for debug storage (.tts) files (default is 1MB)
-#   ibistesttool.maxFileSize=1MB
+# Maximum file size for debug storage (.tts) files (default is 10MB)
+#   ibistesttool.maxFileSize=10MB
```
Comment (nielsm5) [inline core/src/main/resources/AppConstants.properties:415]: Deze dus terug

File: core/src/main/resources/log4j4ibis.properties [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 # maximum file size of xml log files
-log.xml.maxFileSize=1MB
+log.xml.maxFileSize=10MB
 log.xml.maxBackupIndex=9
 
 # maximum file size of msg log files
-log.msg.maxFileSize=1MB
+log.msg.maxFileSize=10MB
 log.msg.maxBackupIndex=9
```
Comment (nielsm5) [inline core/src/main/resources/log4j4ibis.properties:13]: de xml size mag weg, en ik denk toch dat het beter is om hem op 1MB te laten.
In de FrankLogConfigurationFactory#setLevel kan de default naar INFO

**Review by nielsm5 — CHANGES_REQUESTED** [review]

File: core/src/test/java/org/frankframework/logging/FrankLogConfigurationFactoryTest.java [hunk 3]
```diff
@@ -70,13 +70,13 @@ void testPopulateSubstitutions() {
 		properties.setProperty("security.log.level", "${global.log.level}");
 		properties.setProperty("global.log.level", "WARN");
 		properties.setProperty("config.log.level", "INFO");
-		properties.setProperty("log.maxFileSize", "1MB");
+		properties.setProperty("log.maxFileSize", "10MB");
```
[the remark this answers is not in the document]
Comment (jkosternl) [reply core/src/test/java/org/frankframework/logging/FrankLogConfigurationFactoryTest.java:73]: Klopt. Had het overal hetzelfde gemaakt, qua consistentie.

File: ladybug/src/main/resources/log4j4testtool.properties [hunk 4]
```diff
@@ -8,7 +8,7 @@ log4j.appender.TestToolRollingFileAppender=org.apache.log4j.RollingFileAppender
 log4j.appender.TestToolRollingFileAppender.layout=org.apache.log4j.EnhancedPatternLayout
 log4j.appender.TestToolRollingFileAppender.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%t] %c %m%n
 log4j.appender.TestToolRollingFileAppender.File=${log.dir}/testtool4${instance.name.lc}.log
-log4j.appender.TestToolRollingFileAppender.MaxFileSize=1MB
+log4j.appender.TestToolRollingFileAppender.MaxFileSize=10MB
```
[the remark this answers is not in the document]
Comment (jkosternl) [reply ladybug/src/main/resources/log4j4testtool.properties:11]: Oke, zet ik terug.

File: core/src/test/resources/log4j4ibis.properties [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
 # maximum file size of plain log files, can also be KB, e.g. 512KB
-log.maxFileSize=1MB
+log.maxFileSize=10MB
 # maximum number of history logfiles
 log.maxBackupIndex=9
 
 # maximum file size of xml log files
-log.xml.maxFileSize=1MB
+log.xml.maxFileSize=10MB
 log.xml.maxBackupIndex=9
```
[the remark this answers is not in the document]
Comment (jkosternl) [reply core/src/test/resources/log4j4ibis.properties:9]: Oke, haal ik weg.

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**